src/sentry/api/serializers/models/team.py

```diff
@@ -187,20 +197,43 @@ def serialize(
         return d
 
 
+def get_scim_teams_members(
+    team_list: Sequence[Team],
+) -> MutableMapping[Team, MutableSequence[MutableMapping[str, Any]]]:
+    members = RangeQuerySetWrapper(
+        OrganizationMember.objects.filter(teams__in=team_list).select_related("user"),
+        limit=10000,
+    )
+    member_map: MutableMapping[Team, MutableSequence[MutableMapping[str, Any]]] = defaultdict(list)
+    for member in members:
+        for team in member.teams.all():
+            member_map[team].append({"value": str(member.id), "display": member.get_email()})
+    return member_map
+
+
 class TeamSCIMSerializer(Serializer):  # type: ignore
+    def get_attrs(
+        self, item_list: Sequence[Team], user: Any, **kwargs: Any
+    ) -> MutableMapping[Team, MutableMapping[str, Any]]:
+        result: MutableMapping[Team, MutableMapping[str, Any]] = {}
+        for team in item_list:
+            result[team] = {"members": None}
+
+        # For a patch request, we don't need to return the full list of members
+        # so look at exclude_members arg
+        if not kwargs.get("exclude_members", None):
+            member_map = get_scim_teams_members(item_list)
+            for team in item_list:
+                result[team]["members"] = member_map.get(team, [])
+        return result
+
     def serialize(
         self, obj: Team, attrs: Mapping[str, Any], user: Any, **kwargs: Any
     ) -> MutableMapping[str, JSONData]:
         return {
             "schemas": [SCIM_SCHEMA_GROUP],
-            "id": obj.id,
+            "id": str(obj.id),
```

Comment:
changed the id here to be a string -- our IDPs didn't seem to care that it was an int but being consistent with what i've seen other implementations do.